### PR TITLE
[MAINT] fix flake8

### PR DESCRIPTION
**Pull request by Remi-Gau**
fix flake8 errors appearng with the latest flake8 version

### CODE REVIEW

File: nilearn/maskers/tests/test_masker_validation.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def test_check_embedded_nifti_masker():
         masker = _check_embedded_nifti_masker(
             owner, multi_subject=multi_subject
         )
-        assert type(masker) == type(mask)
+        assert type(masker) == type(mask)  # noqa : E721
```
Comment (Remi-Gau) [inline nilearn/maskers/tests/test_masker_validation.py:91]: am unsure how to best fix this one

Comment (ymzayek) [reply nilearn/maskers/tests/test_masker_validation.py:91]: Isn't that equivalent to `assert isinstance(masker, type(mask))`?

**Review by ymzayek — COMMENTED** [review]
LGTM otherwise!

File: nilearn/maskers/tests/test_masker_validation.py (hunk 1 above)
Comment (Remi-Gau) [reply nilearn/maskers/tests/test_masker_validation.py]: that incantation pacified the linting gods on my machine, let's see if this works on the remote gods of the CI too

**Review by bthirion — COMMENTED** [review]
LGTM so far.

File: nilearn/maskers/tests/test_masker_validation.py (hunk 1 above)
Comment (Remi-Gau) [reply nilearn/maskers/tests/test_masker_validation.py]: fixed
the gods are happy

**Pull request merged**